.github/workflows/publish-commit-bottles.yml

```diff
@@ -244,6 +244,7 @@ jobs:
       volumes:
         - /mnt:/mnt
     permissions:
+      attestations: write # for `generate build provenance`
       id-token: write # for `generate build provenance`
       contents: write # for `generate build provenance`
       packages: write # for `generate build provenance`
```

Comment:
Are the last two still needed? Don't see it in the documentation anymore (https://github.com/github-early-access/generate-build-provenance/commit/205cbc2e63a570c31ee710c84e0b2b906f19b8f5)

Reply:
Yeah, I believe that's right, removing.